SecretSanta/src/SecretSanta.Business/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace SecretSanta.Business
+{
+    public class User
+    {
+        public int Id { get; private set; }
```

Comment:
Property should be readonly.
```suggestion
        public int Id { get; }
```